### PR TITLE
New Traps OOB playbooks

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18384

## Description
Added `Traps Quarantine Event` and `Traps Isolate Endpoint` playbooks. (The other two were added last release.)

## Must have
- [x] Tests (No tests - irreverseble commands)
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
@kirbles19 
- [ ] [YAML 1 file](https://github.com/demisto/content/blob/79c237cbdad13419aaae73780aef8920f0d8007f/Playbooks/playbook-Traps_Isolate_Endpoint.yml)
- [ ] [CHANGELOG 1](https://github.com/demisto/content/blob/79c237cbdad13419aaae73780aef8920f0d8007f/Playbooks/playbook-Traps_Isolate_Endpoint_CHANGELOG.md)
- [ ] [YAML 2 file](https://github.com/demisto/content/blob/79c237cbdad13419aaae73780aef8920f0d8007f/Playbooks/playbook-Traps_Quarantine_Event.yml)
- [ ] [CHANGELOG 2](https://github.com/demisto/content/blob/79c237cbdad13419aaae73780aef8920f0d8007f/Playbooks/playbook-Traps_Quarantine_Event_CHANGELOG.md)

### CODE REVIEW

File: Playbooks/playbook-Traps_Isolate_Endpoint.yml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    type: regular
+    task:
+      id: 40b681a4-91a1-4df0-8dcd-6d40cae0e052
+      version: -1
+      name: traps-endpoint-isolate
+      description: Isolates the specified endpoint.
+      script: Traps|||traps-endpoint-isolate
+      type: regular
+      iscommand: true
+      brand: Traps
```
Comment (yaakovi) [inline Playbooks/playbook-Traps_Isolate_Endpoint.yml]: don't use brand, same for all

File: Playbooks/playbook-Traps_Isolate_Endpoint.yml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      endpoint_id:
+        simple: ${inputs.endpoint_id}
```
Comment (yaakovi) [inline Playbooks/playbook-Traps_Isolate_Endpoint.yml]: convert to complex, looks better in UI.
same for all.

File: Playbooks/playbook-Traps_Isolate_Endpoint.yml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: endpoint_id
+  value: {}
+  required: false
+  description: Endpoint ID
+outputs: []
```
Comment (yaakovi) [inline Playbooks/playbook-Traps_Isolate_Endpoint.yml]: add outputs

File: Playbooks/playbook-Traps_Isolate_Endpoint_CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
```
Comment (yaakovi) [inline Playbooks/playbook-Traps_Isolate_Endpoint_CHANGELOG.md]: no need, remove this

File: Playbooks/playbook-Traps_Quarantine_Event_CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
```
Comment (yaakovi) [inline Playbooks/playbook-Traps_Quarantine_Event_CHANGELOG.md]: no need, remove this file

**Comment by ShahafBenYakir** [comment]
Seems to be duplicate to this https://github.com/demisto/content/pull/4531

**Pull request closed without merging**